CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ All notable changes to this project will be documented in this file.
 ### Changed
 - Pending transactions that no longer can be accepted are removed
 - Order of block transaction acceptance (based on ordinal)
+- Fail joining flow if any step during joining process fails and leave the cluster
```

Comment:
`Fail the joining flow and leave the cluster if any step during the joining process fails`

Reply:
Ok I will rephrase it.